packages/network/package.json

```diff
@@ -13,10 +13,11 @@
     "tracker": "bin/tracker.js"
   },
   "main": "dist/composition.js",
+  "browser": "dist/streamr-network.js",
   "types": "dist/composition.d.ts",
   "scripts": {
     "prepare": "tsc -p tsconfig-build.json",
-    "build": "tsc -p tsconfig-build.json",
+    "build": "tsc -p tsconfig-build.json && webpack",
```

Comment:
This should probably be in both `prepare` and `build`, or one should call the other.